### PR TITLE
Update Usage.md

### PR DESCRIPTION
**Pull request by rain2540**

### CODE REVIEW

File: Usage.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ This was a simple demonstration of Swift for TensorFlow. To see example models w
 
 To use Swift for TensorFlow with Xcode, you must have installed a toolchain from [this page](Installation.md).
 
-1. Open Xcode’s `Preferences`, navigate to `Components > Toolchains`, and select the installed Swift for TensorFlow toolchain. The name of the toolchain should start with "Swift for TensorFlow Development Snapshot".
+1. Open Xcode’s `Preferences`, navigate to `Components > Toolchains`, and select the installed Swift for TensorFlow toolchain. The name of the toolchain should start with "Swift for TensorFlow Development Snapshot" OR "Swift for TensorFlow Release".
```
Comment (rxwei) [inline Usage.md]:
```suggestion
1. Open Xcode’s `Preferences`, navigate to `Components > Toolchains`, and select the installed Swift for TensorFlow toolchain. The name of the toolchain should start with "Swift for TensorFlow Development Snapshot" or "Swift for TensorFlow Release".
```

Comment (rain2540) [reply Usage.md]: Thank you for your suggestion, I have changed "OR" to "or".

**Comment by saeta** [comment]
Hey @rain2540 ! Once you sign the CLA, then re-ping this thread, and we can get your change submitted. Thanks! -Brennan

**Comment by rain2540** [comment]
I signed it!

File: Usage.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ This was a simple demonstration of Swift for TensorFlow. To see example models w
 
 To use Swift for TensorFlow with Xcode, you must have installed a toolchain from [this page](Installation.md).
 
-1. Open Xcode’s `Preferences`, navigate to `Components > Toolchains`, and select the installed Swift for TensorFlow toolchain. The name of the toolchain should start with "Swift for TensorFlow Development Snapshot".
+1. Open Xcode’s `Preferences`, navigate to `Components > Toolchains`, and select the installed Swift for TensorFlow toolchain. The name of the toolchain should start with "Swift for TensorFlow Development Snapshot" or "Swift for TensorFlow Release".
```
Comment (dan-zheng) [inline Usage.md]: Currently, the text reads:
```
The name of the toolchain should start with "Swift for TensorFlow Development Snapshot".
```
This is fully accurate? An earnest question: how is being specific better?
```
The name of the toolchain should start with "Swift for TensorFlow Development Snapshot" or "Swift for TensorFlow Release".
```

**Comment by ematejska** [comment]
Hi,  thank you for following up but a complementary change was already made with https://github.com/tensorflow/swift/pull/238 so closing this pull request.

**Pull request closed without merging**